cmip6_downscaling/config.py

```diff
@@ -87,6 +87,19 @@
             "adapt_max": 20,
             "dask_distributed_worker_resources_taskslots": "1",
         },
+        "gateway": {
+            "storage_prefix": "az://",
+            "storage_options": {
+                'container': 'prefect',
+            },
+            "cluster_name": '',  #
+            "extra_pip_packages": "git+https://github.com/carbonplan/cmip6-downscaling.git@gateway",
+            "image": "carbonplan/cmip6-downscaling-prefect:2022.04.03",
```

Comment:
Could we share the image between the the prefect and gateway? Might be easy to update one and forget the other.

Reply:
totally. we should probably pull a number of these options out to a higher level in the config.